portal/lib/doc_lib.php

```diff
@@ -13,7 +13,7 @@
 
 
 // Will start the (patient) portal OpenEMR session/cookie.
-require_once(dirname(__FILE__) . "/../src/Common/Session/SessionUtil.php");
+require_once(dirname(__FILE__) . "/../../src/Common/Session/SessionUtil.php");
```

Comment:
Is this line a critical fix (ie. are things breaking)? (seeing if it needs to get into next patch)

Reply:
Yep breaks current release so need in patch.